### DISCUSSION TITLE
Gabapentinoids + Opioids

### DISCUSSION THREAD

**Discussion by zelixir25** [TripSit/drugs#81] [Drug Interactions]
While these combinations may be tolerated by those who are using these drugs as prescribed, recreational use of gabapentinoids and opioids increases risk of fatal overdose.

Evoy, K. E., Sadrameli, S., Contreras, J., Covvey, J. R., Peckham, A. M., & Morrison, M. D. (2021). Abuse and misuse of pregabalin and gabapentin: A systematic review update. Drugs, 81(1), 125–156. https://doi.org/10.1007/s40265-020-01432-7

Lyndon, A., Audrey, S., Wells, C., Burnell, E. S., Ingle, S., Hill, R., Hickman, M., & Henderson, G. (2017). Risk to heroin users of polydrug use of pregabalin or gabapentin. Addiction, 112(9), 1580–1589. https://doi.org/10.1111/add.13843

**Comment by utaninja** [comment]
Great sources.


Gabapentin specific: Peckham, A.M., Fairman, K.A. & Sclar, D.A. All-Cause and Drug-Related Medical Events Associated with Overuse of Gabapentin and/or Opioid Medications: A Retrospective Cohort Analysis of a Commercially Insured US Population. Drug Saf 41, 213–228 (2018). https://doi.org/10.1007/s40264-017-0595-1
Gomes, Tara, et al. “Gabapentin, Opioids, and the Risk of Opioid-Related Death: A Population-Based Nested Case–Control Study.” PLOS Medicine, vol. 14, no. 10, Oct. 2017, p. e1002396, https://doi.org/10.1371/journal.pmed.1002396.

Pregabalin: 
Elliott, Simon P., et al. “Determining the Toxicological Significance of Pregabalin in Fatalities.” Journal of Forensic Sciences, vol. 62, no. 1, 2017, pp. 169–73, https://doi.org/10.1111/1556-4029.13263.

**Comment by utaninja** [comment]
Interactions:
Gabapentin + Opioids
Pregabalin + Opioids

Central nervous system and/or respiratory-depressant effects may be additively or synergistically present. The two substances potentiate each other strongly and unpredictably, very rapidly leading to unconsciousness. While unconscious, vomit aspiration is a risk if not placed in the recovery position Blackouts/memory loss likely
Both substances potentiate the ataxia and sedation caused by the other and can lead to unexpected loss of consciousness at high doses. Place affected patients in the recovery position to prevent vomit aspiration from excess. Memory blackouts are likely

I'd presume some mixture of both of those messages for the interaction. Do you think we should classify this interaction as dangerous or unsafe? (will change the wording)

**Reply by zelixir25** [reply]
I think we should go with Dangerous similar to all the other CNS depressant combo classifications.

**Reply by utaninja** [reply]
Agreed. Pushing

**Comment by utaninja** [comment]
Pushed to https://github.com/TripSit/drugs/issues/111
